### ISSUE TITLE
Training loss never achieves < 0.5 loss

### ISSUE THREAD

**Issue by ghost** [Rayhane-mamah/Tacotron-2#334]
Hi all,

Hope you can help a novice understand what he's doing wrong.

I've been trying to train a custom dataset which I put together myself (about 600 audio samples). To make it easy I used the metadata file from LJSpeech so that the pre-processing should work without modification. 

The probelm I've experienced is that the loss quickly reached about 0.5% +- 0.05% however for the last two days the loss has bounced between 0.45 and 0.55 and never seems to dip any lower. Is it possible that the training has become stuck in a local optima and is never getting out? Or is perhaps due to the small number of audio samples I've provided. I've included the log file [here](https://pastebin.com/70T3F7Ce) heavily edited to get the file under 512k!!.

I'd appreciate any suggestions... if you get in touch I can send  the results of the latest synthesis.

Cheers

**Comment by swong28** [comment]
Can you provide your pre-trained model? my computer is really slow and unable to train the model quickly. Thank you.

**Comment by GregCKrause** [comment]
It could be that it is stuck in a local optima.

If you're using a custom dataset, you may benefit from a transfer learning approach as described in  #279.
